packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -96,10 +102,13 @@ $side-padding: 1.2em;
 
 /** @define statement; weak */
 .statement {
+    .table__head {
+        font-weight: bold;
+    }
+
```

Comment:
Is this `.statement` different from the `statement.css` file that was created? Can we move the styles there?